src/main/scala/dsptools/Chisel3TestersCompatibility.scala

```diff
@@ -0,0 +1,14 @@
+package chisel3.iotesters
```

Comment:
From a research standpoint, one of the goals was reusability of RTL-level components. While industry is better at engineering and building things that last (or maybe not, considering the widespread use of Perl to generate Verilog), if we had to rewrite Chisel every time it got too complex, then we really haven't accomplished easy re-use. And it would be a massive waste of time, not just for infrastructure builders, but for users who waste hours / days working through obscure bugs from complex unforeseen interactions.

There's also a trade-off between writing your one-of solution that basically needs to get thrown away, and being a part in building infrastructure. If everyone did the former, then everyone would be starting from near-scratch (since the last person's code doesn't work anymore / isn't structurally sound to build upon / isn't documented / is basically on fire) rather than being able to build on top of existing projects and systems.

If the tape-out deadlines are so tight that developers feel they are pressured into building hax on top of hax, that seems like a problem with the deadlines, possibly with misconceptions of how stable / how powerful the infrastructure is in the setting of deadlines. Infrastructure is (hopefully) the kind of stuff that gets build / fixed once and can be re-used in future projects, but hax upon hax are a recurring cost. And it usually gets worse each time.